compact/materials.xml

```diff
@@ -20,6 +20,10 @@
     <fraction n="0.234" ref="O"/>
     <fraction n="0.012" ref="Ar"/>
   </material>
+  <material name="HeGas">
+    <D type="density" unit="g/cm3" value="0.0000838"/>
```

Comment:
This seems like the wrong density for He at STP. Should be about a tenth of air, defined above, but this is another 100 times smaller. Is this not at STP? Can we add pressure in the name, e.g. `HeGas0.001bar`.